### DISCUSSION TITLE
Last Updated On Date

### DISCUSSION THREAD

**Discussion by cassievann** [CMSgov/hospital-price-transparency#101] [Q&A] [answered]
For the CSV Wide Format, should the last updated on date be the date that we pulled the file or the actual date charges were last updated?

**Comment by RidenourK** [comment, marked as answer]
@cassievan. Thank you for your question. 

As indicated at [45 CFR 180.50(b)(2)(i)(B),](https://www.ecfr.gov/current/title-45/part-180#p-180.50(b)(2)(i)(B)) a hospital must encode in its machine-readable file all standard charge information, including the date of most recent update **to the standard charge information** in the machine-readable file.
